spec/e2e/enum_spec.rb

```diff
@@ -70,6 +70,16 @@
       expect(ids).to include(file2.id)
     end
 
+    it 'supports arrays' do
```

Comment:
As I understand the rspec way, things should be separated into examples as much as possible.  Obviously you need to balance that if there's a lot of setup.